### PR TITLE
feature: support props.wrapStyle

### PR DESCRIPTION
**Pull request by harbin1053020115**
Support props.wrapStyle to add styles to wrap domElement

### CODE REVIEW

File: src/parcel.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import {SingleSpaContext} from '../lib/single-spa-react.js'
 export default class Parcel extends React.Component {
   static defaultProps = {
     wrapWith: 'div',
+    wrapStyle: {},
```
Comment (harbin1053020115) [inline src/parcel.js]: default props.wrapStyle

Comment (harbin1053020115) [reply src/parcel.js]: Default set nothing to wrap domElement

File: src/parcel.js [hunk 2]
```diff
@@ -91,7 +95,7 @@ export default class Parcel extends React.Component {
         )
         : undefined
 
-      return React.createElement(this.props.wrapWith, {ref: this.handleRef}, children)
+      return React.createElement(this.props.wrapWith, {ref: this.handleRef, style: this.props.wrapStyle}, children)
```
Comment (harbin1053020115) [inline src/parcel.js]: Pass props.wrapStyle to domElement

File: src/parcel.js [hunk 3]
```diff
@@ -36,6 +37,9 @@ export default class Parcel extends React.Component {
         domElement = this.el
       } else {
         this.createdDomElement = domElement = document.createElement(this.props.wrapWith)
+        Object.keys(this.props.wrapStyle).forEach(key => {
```
Comment (harbin1053020115) [inline src/parcel.js]: Set domElement stlyes

File: src/parcel.test.js [hunk 4]
```diff
@@ -43,6 +43,11 @@ describe(`<Parcel />`, () => {
     expect(wrapper.find('div').length).toBe(1)
   })
 
+  it(`renders a div wrap with style`, () => {
```
Comment (harbin1053020115) [inline src/parcel.test.js]: Check if style attributes is collect

**Comment by joeldenning** [comment]
Thanks for creating this pull request. At first glance this looks good, and like a sensible option to add. Will do a deeper review in the next day or two so we can get this merged and published.

**Comment by joeldenning** [comment]
Released in https://github.com/single-spa/single-spa-react/releases/tag/v2.12.0. Could you please submit a pull request to the documentation for this? The file to change is https://github.com/single-spa/single-spa.js.org/blob/master/website/versioned_docs/version-5.x/ecosystem-react.md. Thanks for your help with this.

**Comment by harbin1053020115** [comment]
> Released in https://github.com/single-spa/single-spa-react/releases/tag/v2.12.0. Could you please submit a pull request to the documentation for this? The file to change is https://github.com/single-spa/single-spa.js.org/blob/master/website/versioned_docs/version-5.x/ecosystem-react.md. Thanks for your help with this.

Pls check this MR: https://github.com/single-spa/single-spa.js.org/pull/211

**Pull request merged**